extensions/amp-app-banner/0.1/amp-app-banner.js

```diff
@@ -435,4 +435,6 @@ function updateViewportPadding(state) {
 }
 
 
-AMP.registerElement('amp-app-banner', AmpAppBanner, CSS);
+AMP.extension('amp-app-banner', function(AMP) {
+  AMP.registerElement('amp-app-banner', AmpAppBanner, CSS);
```

Comment:
Yeah. It's definitely annoying. This really means our current `AMP.push({n: 'amp-app-banner', f: ...})`